### DISCUSSION TITLE
[V5] unlink file after response

### DISCUSSION THREAD

**Discussion by Pratikkshirsagar** [adonisjs/core#3401] [Help]
Hello,
How can I unlink file after send the file using ```response.attachment()```  or ``` response.download()```.

**Comment by feniljariwala82** [comment]
Unlink in terms of what? Do you want to delete the file after sending as a response to the client?

**Comment by holoyan** [comment]
@feniljariwala82 I'm having same issue, I want to delete file after sending response

https://github.com/adonisjs/core/discussions/4341
